History.md

```diff
@@ -1,3 +1,8 @@
+## 5.2.2 / 2021-02-15
+
+* Bugfixes
+  * Remove `**` from `options` parameter ([#2555])
+
 ## 5.2.2 / 2021-02-
 
 * Bugfixes
```

Comment:
this was causing puma to throw argument error when running sinatra with shotgun
removing double splat from options param fixes the issue, shogun runs fine with puma and sinatra
all tests still passing after making the fix